CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/processing/GraphRecordStoreUtilities.java

```diff
@@ -64,6 +64,10 @@ private GraphRecordStoreUtilities() {
     public static final String DIRECTED_KEY = "[directed]<string>";
     public static final String COMPLETE_WITH_SCHEMA_KEY = "[complete_with_schema]<string>";
     public static final String DELETE_KEY = "[delete]<string>";
+    
+    private static final String SELECTED_ATTRIBUTE = "selected";
```

Comment:
VisualConcept.VertexAttributes.Selected.getName()

Reply:
Doing this would introduce a cyclic dependency

Reply:
then just rename it to SELECTED_ATTRIBUTE_LABEL :)

Reply:
Don't forget this one :)

Reply:
Done